### PR TITLE
IndieWeb metadata to pump.io posts

### PR DESCRIPTION
**Pull request by vxcamiloxv**
Closes: #1293

### CODE REVIEW

File: public/template/profile-block.jade [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
           = profile.preferredUsername
 
     if profile.location
-      p.location!= profile.location.displayName
+      p.location.p-locality!= profile.location.displayName
     if profile.summary
-      p.summary!= profile.summary
-    // TODO: HOME PAGE HERE
+      p.summary.p-note!= profile.summary
+    // a.u-url(href="HOME PAGE")!=TODO: HOME PAGE HERE
```
Comment (strugee) [inline public/template/profile-block.jade]: This should probably be `=`, not `!=`

Comment (vxcamiloxv) [inline public/template/profile-block.jade]: Made

File: public/template/profile-block.jade [hunk 2]
```diff
@@ -21,4 +21,4 @@
       p.location.p-locality!= profile.location.displayName
     if profile.summary
       p.summary.p-note!= profile.summary
-    // a.u-url(href="HOME PAGE")!=TODO: HOME PAGE HERE
+    // a.u-url(href="HOME PAGE")=TODO: HOME PAGE HERE
```
Comment (strugee) [inline public/template/profile-block.jade]: Can you also add a space after the `=`? Sorry I didn't catch that earlier.

Comment (vxcamiloxv) [inline public/template/profile-block.jade]: made

**Comment by strugee** [comment]
Thank you!

**Pull request merged**